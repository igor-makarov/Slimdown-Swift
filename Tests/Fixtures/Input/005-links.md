A [link](http://www.google.com) and [another link](http://yahoo.com/).