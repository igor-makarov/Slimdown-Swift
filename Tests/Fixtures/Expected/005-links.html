<p>A <a href='http://www.google.com'>link</a> and <a href='http://yahoo.com/'>another link</a>.</p>